chore: copy updates for help page - AI-492

JBR90 opened the pull request:
## Description

- Adds the copy changes from this [ticket](https://www.notion.so/oaknationalacademy/Updates-to-help-page-5aad6523583549148dd7bdfbcc8edfa0)


## How to test

https://oak-ai-lesson-assistant-9bde9x93r.vercel-preview.thenational.academy/aila/help


## Checklist

- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Does this PR update a package with a breaking change

mikeritson-oak commented on apps/nextjs/src/app/aila/help/index.tsx:
```diff
@@ -204,8 +207,9 @@ const Help = () => {
               lesson plan (which includes all sections), editable slides and
-              starter and exit quizzes. You can also share your lesson as a
-              link, which can be viewed online.
+              starter and exit quizzes, a worksheet with your practice tasks and
```
I feel like this new paragraph is a bit clunky and has at least one too many 'and' in there, possibly this one:
editable slides and starter and exit quizzes, as we then use another one at the end of the sentence. May just need confirmation of how we want the actual section to read. Seems like it could be an intro sentence, list and then sharing outro sentence, but it also kind of replicates the list that is already up above. @hannah-bethclark

mikeritson-oak commented on apps/nextjs/src/app/aila/help/index.tsx:
```diff
@@ -187,7 +190,7 @@ const Help = () => {
             </p>
             <ul className="mb-6 list-disc pl-12">
-              <li>adapt the content for a specific reading age.</li>
+              <li>increase or decrease the literacy level of your keywords</li>
```
This addition doesn't have a full stop at the end of it whilst all other items in this list do. I'm sure the general rule is that items in lists should not have full stops, so it might be that the other items are wrong, not this one.

mikeritson-oak reviewed (changes requested):
In the final section `AI and accuracy` I don't think Retrieval augmented generation should be capitalised, if we want to adhere to the sentence case policy.
(I couldn't find how to comment on this inline with the code.)

JBR90 commented:
> Two general notes from testing:
> 
> 1. I noticed that all of the headings we now have are not reflected in the menu structure on the left, is this intentional? I'd assume they should all be present.
> 2. Clicking on the links in this list of section doesn't take the viewport to the selected section. There seems to be some movement but only to the top of the content.
> 
> Double checked on production and both are these are happening there, so not necessarily due to these changes but in this area.

Have address the comments, fixed the nav and added missing sections and refactored everything to Oak Components. Will await @hannah-bethclark feedback for any copy changes

stefl commented on apps/nextjs/src/app/aila/help/index.tsx:
```diff
@@ -30,196 +44,247 @@ export const HelpContent = () => {
+                </OakP>
+                <OakUL className="mb-6 list-disc pl-12">
+                  <OakLI>Learning outcome</OakLI>
+                  <OakLI>Learning cycle outcomes</OakLI>
```
This seems different to the language in the app

JBR90 replied:
Yeah good spot, have adjusted to match current lesson sections

stefl commented on apps/nextjs/src/app/aila/help/index.tsx:
```diff
@@ -30,196 +44,247 @@ export const HelpContent = () => {
-                >
+                </OakHeading>
+                <OakP $mb={"space-between-xs"}>
+                  Once you’ve completed all the sections in the lesson, you’ll
```
Should this be &apos; instead of a '?

codeincontext replied:
I think it's a `’` rather than `'`, which is correct (and would be an &apos; symbol in raw html)

codeincontext reviewed (changes requested):
Bringing the layout into the component has added the layout to the storybook story, which we don't normally do. If you look at other pages you'll see a pattern to rename this component to `HelpContent` and wrap it with a component which includes the layout

mikeritson-oak reviewed (approved):
Changes have all been made, we have one question around copy outstanding that will be raised in a new ticket. 👍

oak-machine-user commented:
:tada: This PR is included in version 1.17.0 :tada:

The release is available on [GitHub release](https://github.com/oaknational/oak-ai-lesson-assistant/releases/tag/v1.17.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
